miniapps/electromagnetics/tesla.cpp

```diff
@@ -35,6 +35,10 @@
 //   A cylindrical bar magnet in a metal sphere:
 //      mpirun -np 4 tesla -bm '0 -0.5 0 0 0.5 0 0.2 1'
 //
+//   A Halbach array of permanent magnets:
```

Comment:
This is a duplicate, see line 48

Reply:
The first run uses such a coarse mesh that it would not be very interesting anyway.  Good catch!